eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -19,7 +19,7 @@ function Get-ChangeLogEntries {
     LogError "ChangeLog[${ChangeLogLocation}] does not exist"
     return $null
   }
-  LogDebug "Extracting entries from [${ChangeLogLocation}]."
+  Write-Verbose "Extracting entries from [${ChangeLogLocation}]."
```

Comment:
Should we just add a LogVerbose?

Reply:
This is a PS specific form of logging. The logging functions are mostly to abstract between local, devops, and github actions. So, I don't think that is needed. Perhaps we need another level eventually.